README.md

```diff
@@ -62,12 +62,6 @@
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- Live Demo -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
-
-<a href="https://izebiafe.github.io/awesome-book"> Coming soon</a>
-
 <!-- GETTING STARTED -->
```

Comment:
- [ ] The link to your MIT LICENSE is broken and displays a 404 error page as shown below:

![image](https://github.com/ismayito/Awesome-Books/assets/77106962/c06130bd-0324-442c-99b9-5c9cc0877f8a)

Kindly fix the path by using `./MIT.md` instead of `./MIT`. 👍 😎